test/user-test.js

```diff
@@ -3,57 +3,137 @@ import { expect } from 'chai';
   });
 
-  it('Should have a property of favoriteRecipes with a default value', () => {
-    expect(user1.favoriteRecipes).to.eql([]);
+  it('Should be able to add multiple recipes to favoriteRecipes', function() {
+    user1.addToFavorites(recipeData[0]);
+    user1.addToFavorites(recipeData[1]);
+
+    expect(user1.favoriteRecipes).to.deep.equal([recipeData[0], recipeData[1]]);
   });
 
-  it('Should be able to add recipes to favoriteRecipes', () =>{
-    user1.addToFavorites(recipeData[0])
-    expect(user1.favoriteRecipes.includes(recipeData[0])).to.eql(true);
+  it('Should be able to remove recipes from favoriteRecipes', function() {
+    user1.addToFavorites(recipeData[0]);
+    user1.addToFavorites(recipeData[1]);
+    user1.addToFavorites(recipeData[2]);
+    user1.removeFromFavorites(recipeData[1]);
+
+    expect(user1.favoriteRecipes).to.deep.equal([recipeData[0], recipeData[2]]);
+  });
+
+  it('Should be able to filter through favoriteRecipes by tag', function() {
+    user1.addToFavorites(recipeData[0]);
+    user1.addToFavorites(recipeData[1]);
+
+    expect(user1.filterFavorites('antipasti')).to.deep.equal([recipeData[0]]);
   });
 
-  it('Should be able to remove recipes from favoriteRecipes', () =>{
-    user1.removeFromFavorites(recipeData);
-    expect(user1.favoriteRecipes).to.eql([]);
+  it('Should be able to search favoriteRecipes ingredient', function(){
+    user1.addToFavorites(recipeData[0]);
+    user1.addToFavorites(recipeData[1]);
+
+    expect(user1.findFavorites('egg')).to.deep.equal([recipeData[0]]);
   });
 
-  it('Should be able to filter through favoriteRecipes by tag', () => {
+  it('Should be able to return multiple search of favoriteRecipes by ingredient'
+    , function(){
+      user1.addToFavorites(recipeData[0]);
+      user1.addToFavorites(recipeData[1]);
+      user1.addToFavorites(recipeData[5]);
+
+      expect(user1.findFavorites('egg')).to.deep.equal([recipeData[0],
+        recipeData[5]]);
+    });
+
+  it('Should be able to search favoriteRecipes by name', function(){
     user1.addToFavorites(recipeData[0]);
     user1.addToFavorites(recipeData[1]);
-    expect(user1.filterFavorites('antipasti')).to.eql([recipeData[0]]);
+
+    expect(user1.findFavorites('Chocolate')).to.deep.equal([recipeData[0]]);
   });
 
-  it('Should be able to search favoriteRecipes by name or ingredient', () => {
+  it('Should be able to search multiple favoriteRecipes by name', function(){
     user1.addToFavorites(recipeData[0]);
     user1.addToFavorites(recipeData[1]);
-    expect(user1.findFavorites('egg')).to.eql([recipeData[0]]);
+    user1.addToFavorites(recipeData[3]);
+    user1.addToFavorites(recipeData[6]);
+
+    expect(user1.findFavorites('Chocolate')).to.deep.equal([recipeData[0],
+      recipeData[6]]);
   });
 
-  it('Should be able to check ingredients in User/s pantry for a given recipe', () => {
+  it.skip('Should be able to check ingredients in User/s pantry for a given recipe', () => {
```

Comment:
For this test and the one below it,  we should change the last part of the it line to be function () { since that's what was used for all the tests above it.

Reply:
Ok. I figured that we will need to change the function set up so that it will align with the ES5 format and it should stay consistent throughout the projet.